addon/components/o-s-s/progress-bar.hbs

```diff
@@ -0,0 +1,19 @@
+<div class="oss-progress-bar--wrapper">
+  {{#if @label}}
+    <span class="oss-progress-bar--label">
+      {{@label}}
+    </span>
+  {{/if}}
```

Comment:
Are we sure we want to handle the label in the component directly ? 🤔 
@phndiaye WDYT ?